src/Lykke.Service.OrderBookAnalysis.Services/OrderBookConverter.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Linq;
+using AutoMapper;
+using Lykke.Common.ExchangeAdapter.Contracts;
+using Lykke.Service.OrderBookAnalysis.AzureRepositories;
+using Lykke.Service.OrderBookAnalysis.Contracts;
+
+namespace Lykke.Service.OrderBookAnalysis.Services
+{
+    public static class OrderBookConverter
+    {
+        public static OrderBookSnapshot FromOrderBook(DateTime ts, OrderBook orderBook)
+        {
+            var mid = (orderBook.BestAskPrice + orderBook.BestAskPrice) / 2;
```

Comment:
Should it be (BestAskPrice + BestBidPrice) / 2 ?

Reply:
fixed + unit test has been written